Pokemon App/Pokemon App/CardsCollectionView/CardCollectionViewCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  CardCollectionViewCell.swift
+//  Pokemon App
+//
+//  Created by Novan Agung Waskito on 15/11/22.
+//
+
+import UIKit
+
+class CardCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var pokemonCard: UIImageView!
+    override func awakeFromNib() {
```

Comment:
Indentationnya tolong rapihin dan penamaan variabel tolong yang lebih deskriptif
```suggestion
    @IBOutlet weak var pokemonImageView: UIImageView!
    
    override func awakeFromNib() {
```

Reply:
oh okay itu pokemonCardImageView gitu ya biar lebih jelas